opensrp-app/src/main/java/org/smartregister/repository/LocationTagRepository.java

```diff
@@ -0,0 +1,141 @@
+package org.smartregister.repository;
+
+import android.content.ContentValues;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import net.sqlcipher.Cursor;
+import net.sqlcipher.database.SQLiteDatabase;
+
+import org.smartregister.domain.LocationProperty;
+import org.smartregister.domain.LocationTag;
+import org.smartregister.util.PropertiesConverter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import timber.log.Timber;
+
+/**
+ * Created by ilakozejumanne on 01/29/20.
+ */
+public class LocationTagRepository extends BaseRepository {
+
+    protected static final String NAME = "name";
+    protected static final String LOCATION_ID = "location_id";
+    protected static final String LOCATION_TAG_TABLE = "location_tag";
+    protected static final String[] COLUMNS = new String[]{NAME,LOCATION_ID};
+    private static final String CREATE_LOCATION_TAG_TABLE =
+            "CREATE TABLE " + LOCATION_TAG_TABLE + " (" +
+                    NAME + " VARCHAR NOT NULL, " +
+                    LOCATION_ID + " VARCHAR NOT NULL, " +
+                    "PRIMARY KEY ("+NAME+", "+LOCATION_ID+")) ";
+    private static final String CREATE_LOCATION_TAG_NAME_INDEX = "CREATE INDEX "
+            + LOCATION_TAG_TABLE + "_" + NAME + "_ind ON " + LOCATION_TAG_TABLE + "(" + NAME + ")";
```

Comment:
This is not required, there are no queries that are querying with the name, all the queries filter using the `location_Id` column.

So if you want to retain the index define it on the `location_Id` column, although index is not required since `location_Id` is part of the primary key.